datastore/writer/postgresql_backend/sql_database_backend_service.py

```diff
@@ -208,6 +208,14 @@ def write_model_updates_action_worker(self, models: Dict[Fqid, Model]) -> None:
             use_execute_values=True,
         )
 
+    def write_model_deletes_action_worker(self, fqids: List[Fqid]) -> None:
+        """Physically delete of action_workers"""
+        statement = dedent(
+            """\
+            delete from models where fqid in %s;"""
+        )
```

Comment:
Why not use "delete from models where fqid in %s;" here?

Reply:
done